pkg/resourcecreator/pod/pod.go

```diff
@@ -20,11 +20,12 @@ import (
 )
 
 const (
-	naisAppNameEnv     = "NAIS_APP_NAME"
-	naisNamespaceEnv   = "NAIS_NAMESPACE"
 	naisAppImageEnv    = "NAIS_APP_IMAGE"
-	naisClusterNameEnv = "NAIS_CLUSTER_NAME"
+	naisAppNameEnv     = "NAIS_APP_NAME"
 	naisClientId       = "NAIS_CLIENT_ID"
+	naisClusterNameEnv = "NAIS_CLUSTER_NAME"
+	naisNamespaceEnv   = "NAIS_NAMESPACE"
+	defaultPort        = "8080"
```

Comment:
Som @Kyrremann påpeker, så definerer vi allerede default port i liberator, syns ikke vi skal ha flere definisjoner rundt omkring, det er å be om trøbbel senere.

Reply:
i testene var ikke defaultene applyet, selv om pod.go skjer etter ApplyDefault, så må grave litt i hvorfor